compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -628,13 +671,6 @@ execLint inputFiles opts =
            withPackageConfig defaultProjectPath $ \PackageConfigFields {pSrc} -> do
            getDamlRootFiles pSrc
        fs -> forM fs $ fmap toNormalizedFilePath' . relativize
-     setDlintDataDir :: Options -> IO Options
-     setDlintDataDir opts = do
-       defaultDir <-locateRunfiles $
-         mainWorkspace </> "compiler/damlc/daml-ide-core"
-       return $ case optDlintUsage opts of
-         DlintEnabled _ _ -> opts
-         DlintDisabled  -> opts{optDlintUsage=DlintEnabled defaultDir True}
```

Comment:
now that `cmdLint` only allows the user to change the configuration of the linter, but not to turn it off, we don't need to hardcode these